src/callinput.c

```diff
@@ -210,6 +213,26 @@ char callinput(void)
 		break;
 	    }
 
+	case 17:	// CTRL+q
+	    {
+		if (qtcdirection & 1) {
+		    qtc_main_panel(RECV);
+		}
+		if (qtcdirection == 2) {
+		    qtc_main_panel(SEND);
+		}
```

Comment:
What happens if qtcdirection is set to 3 (BOTH)?
Do we show the qtc panel twice after each other?